modules/paging.py

```diff
@@ -0,0 +1,354 @@
+import logging
+import asyncio
+from textwrap import wrap
+
+from modules.custom_embed import default_embed
+from modules.emoji_utils import ri_at_index, ri_alphabet, index_of_ri, normalize
+
+from emoji import emojize
+
+logger = logging.getLogger('voluspa.module.paging')
+
+class _MenuBase:
```

Comment:
Minor note, PEP 8 -- stylistically, you'll want two spaces before a class definition. :+1: